packages/components/src/components/input/input.styles.ts

```diff
@@ -298,13 +282,13 @@ export default css`
   }
 
   /* Hide the built-in number spinner */
-  .input--no-spin-buttons input[type='number']::-webkit-outer-spin-button,
-  .input--no-spin-buttons input[type='number']::-webkit-inner-spin-button {
+  .input--no-browser-spin-buttons input[type='number']::-webkit-outer-spin-button,
+  .input--no-browser-spin-buttons input[type='number']::-webkit-inner-spin-button {
     -webkit-appearance: none;
     display: none;
   }
 
-  .input--no-spin-buttons input[type='number'] {
+  .input--no-browser-spin-buttons input[type='number'] {
     -moz-appearance: textfield;
   }
```

Comment:
Wait. The new buttons should be a replacement for the browser spin buttons. There should be always removed? Do we need a class then at all?

Reply:
Thats the reason I was asking. We should just disable the buttons always if possible. However, shoelace has a prop `no-spin-buttons` which makes it possible to remove the +/- buttons and still use type=number. I think this is the Reason @kirchsuSICKAG added this.

Reply:
Nevermind. I can just remove the class in the css selector and do not need a specific class for this, which is always set true, you`re right

Reply:
I removed the class